Makefile

```diff
@@ -1529,17 +1529,22 @@ cpplint: lint-cpp
 # Try with '--system' if it fails without; the system may have set '--user'
 lint-py-build:
 	$(info Pip installing ruff on $(shell $(PYTHON) --version)...)
-	$(PYTHON) -m pip install --upgrade --target tools/pip/site-packages ruff==0.5.2 || \
-		$(PYTHON) -m pip install --upgrade --system --target tools/pip/site-packages ruff==0.5.2
+	$(PYTHON) -m pip install --upgrade --target tools/pip/site-packages ruff==0.6.0 || \
```

Comment:
0.6.1… it moves so fast!

Reply:
Indeed. A few weeks ago it was `0.5.2`. haha